vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -47,6 +48,9 @@ The subscriptions added to the `IReactInstanceSettings.Notifications` are kept a
     DOC_STRING("Get `ReactDispatcherHelper::JSDispatcherProperty` from the Properties property bag.")
     IReactDispatcher JSDispatcher { get; };
 
+    DOC_STRING("Get the JSI runtime for the running React instance. It can be null if Web debugging is used.")
```

Comment:
I saw a comment above to crash instead, update the doc if so

Reply:
This is different. This retrieves the JsiRuntime from IReactContext. It does not crash. It returns null instead. The crash is in the ExecuteJsi function that supposed to execute user lambda.